tests/test_spin_reversal_transform.py

```diff
@@ -14,15 +14,96 @@
 
 import unittest
 
-from dimod import ExactSolver, RandomSampler, SimulatedAnnealingSampler
+from dwave.system.testing import MockDWaveSampler
+from dimod import NullSampler, ExactSolver, RandomSampler, SimulatedAnnealingSampler
+from greedy import SteepestDescentSolver
 import dimod.testing as dtest
 
 from dwave.preprocessing.composites import SpinReversalTransformComposite
 
+import numpy as np
+
 class TestSpinTransformComposite(unittest.TestCase):
```

Comment:
I think we should add some more tests via `dimod.testing.load_sampler_bqm_tests`, see https://github.com/dwavesystems/dimod/blob/abd3d62655818aecb40f67326476e357fd0bda08/tests/test_spin_transform.py#L28

I think the bug that has it currently commented out as been fixed by this PR.